lib/vmpooler/api.rb

```diff
@@ -168,6 +168,45 @@ def hostname_shorten hostname
           JSON.pretty_generate(result)
         end
 
+        get '/status/?' do
+          content_type :json
+
+          result = {}
+          clone_times = []
```

Comment:
Is this endpoint being used for health and monitoring purposes? Would it be beneficial to have clone times stored with the hostname that was cloned (a hash instead of an array)?